packages/epo-widget-lib/src/hooks/useInterval.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useRef } from "react";
+
+export default function useInterval(
```

Comment:
pulled in from here: https://overreacted.io/making-setinterval-declarative-with-react-hooks/